spec/models/airport_spec.rb

```diff
@@ -1,45 +1,32 @@
 require "rails_helper"
 
 RSpec.describe Airport, type: :model do
-  before(:each) { @airport = create(:airport, flights_count: 2) }
 
-  describe "Initializing Airport" do
-    let(:airport1) { build(:airport, name: nil) }
-    let(:airport2) { build(:airport, country: nil) }
-    let(:airport3) { build(:airport, state: nil) }
-    let(:airport4) { build(:airport, airport_code: nil) }
+  describe "Active model validation" do
+    it { should validate_presence_of(:name) }
```

Comment:
It is recommended to make use of `expect` instead of `should`.

In this case, you'd have this instead:
`it { is_expected.to validate_presence_of(:name) } `

Again, I'd refer you to http://betterspecs.org/#expect. ⚠️